main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -58,7 +58,7 @@ object DocAst {
 
     case class Tuple(elms: List[Expr]) extends Atom
 
-    case class Tag(sym: Symbol.CaseSym, args: List[Expr]) extends Atom
+    case class Tag(sym: Sym, args: List[Expr]) extends Atom
```

Comment:
IMO just change this to be string directly. This AST is meant to be as lightweight and simple as possible. but i like the change

Reply:
Idk, it gives a bit more type safety